libcloud/compute/drivers/openstack.py

```diff
@@ -2979,7 +3012,7 @@ def ex_list_subnets(self):
             self._subnets_url_prefix).object
         return self._to_subnets(response)
 
-    def ex_create_subnet(self, name, network, cidr, ip_version=4):
+    def ex_create_subnet(self, name, network, cidr, ip_version=4, **kwargs):
```

Comment:
I would prefer to explicitly declare support arguments. Using ``**kwargs`` makes it impossible / hard to see which arguments method takes without checking the code. It also makes automatic code introspection impossible.

https://libcloud.readthedocs.io/en/latest/development.html#don-t-abuse-kwargs